tests/e2e_decorrelation/test_scalar_subqueries.py

```
@@ -0,0 +1,661 @@
+            SELECT u.id, u.name,
+                   (SELECT MAX(o.amount) FROM pg.orders o) AS max_order_amount
+            FROM pg.users u
+        """
+
+        parser = Parser()
+        binder = Binder(catalog)
+        decorrelator = Decorrelator()
+
+        logical_plan = parser.parse(sql)
+        bound_plan = binder.bind(logical_plan)
+        decorrelated_plan = decorrelator.decorrelate(bound_plan)
```

Ruff (0.14.5)

Local variable `decorrelated_plan` is assigned to but never used

Remove assignment to unused variable `decorrelated_plan`

(F841)